pkg/mapper/dynamicfile/dynamicfile.go

```diff
@@ -0,0 +1,262 @@
+package dynamicfile
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/aws-iam-authenticator/pkg/config"
+)
+
+type DynamicFileMapStore struct {
+	mutex sync.RWMutex
+	users map[string]config.UserMapping
+	roles map[string]config.RoleMapping
+	// Used as set.
+	awsAccounts map[string]interface{}
```

Comment:
Its common in Kubernetes org projects to use the explicitly defined `sets.String` when a set is desired.  Its a part of k8s.io/apimachinery (https://github.com/kubernetes/apimachinery/blob/master/pkg/util/sets/string.go).  I would recommend it because its more readable.